RealmSwift/LinkingObjects.swift

```diff
@@ -469,7 +509,7 @@ public class LinkingObjectsBase: NSObject, NSFastEnumeration {
     internal let objectClassName: String
     internal let propertyName: String
 
-    private var cachedRLMResults: RLMResults?
+    fileprivate var cachedRLMResults: RLMResults?
```

Comment:
Note that [CI is complaining about the code as-is](https://ci.realm.io/job/objc_pr/configuration=Debug,swift_version=2.2,target=osx-swift/3865/console):
```
/Users/realm/workspace/objc_pr/configuration/Debug/swift_version/2.2/target/osx-swift/RealmSwift/LinkingObjects.swift:512:5: error: expected declaration
    fileprivate var cachedRLMResults: RLMResults?
    ^
/Users/realm/workspace/objc_pr/configuration/Debug/swift_version/2.2/target/osx-swift/RealmSwift/LinkingObjects.swift:512:16: error: consecutive declarations on a line must be separated by ';'
    fileprivate var cachedRLMResults: RLMResults?
               ^
               ;
```